packages/vkui/src/components/Accordion/Accordion.tsx

```diff
@@ -66,9 +66,18 @@ export const Accordion: React.FC<AccordionProps> & {
 };
 
 Accordion.displayName = 'Accordion';
+Object.defineProperty(Accordion, 'name', {
+  value: 'Accordion',
+});
```

Comment:
Можно вот так попробовать, чтобы вырезалось на проде

```suggestion
if (process.env.NODE_ENV === 'development') {
  defineComponentDisplayNames(Accordion, 'Accordion');
}
```

```tsx
export const defineComponentDisplayNames = (Component: AnyFunction, name: string) => {
  const displayName = { value: name };
  Object.defineProperties(Component, { name: displayName, displayName: displayName });
}
```

> Можно было бы спрятать `process.env.NODE_ENV === 'development'` в `defineComponentDisplayNames`, но тогда на проде будет лишний вызов функции на каждый компонент, что так себе
>
> ```tsx
> export const defineComponentDisplayNames =
>  process.env.NODE_ENV === 'development'
>    ? (Component: AnyFunction, name: string) => {
>        const displayName = { value: name };
>        Object.defineProperties(Component, { name: displayName, displayName: displayName });
>      }
>    : noop;
>

Reply:
Добавил defineComponentDisplayNames